client/src/cmdlfindala.c

```diff
@@ -112,6 +112,17 @@ static void decodeHeden2L(uint8_t *bits) {
     PrintAndLogEx(SUCCESS, "    Heden-2L    | %u", cardnumber);
 }
 
+bool parityEven(uint16_t x) {
```

Comment:
Please don't define and expose parity functions in indala src and header files.
Include parity.h
If you need a 16b version, include it there, sth like
```c
static inline uint8_t evenparity16(uint16_t x) {
    x ^= x >> 8;
    return evenparity8(x);
}
```

Reply:
Friendly point of order, parity.c contains a single LUT for 8-bit odd parity only, and a commented out LUT for even parity... which I get, given that they are complimentary, but why a LUT at all? Using it requires loading the entire thing into memory, which is going to take more CPU time than computing parity using XORs, and then there's the memory usage. Granted that's not a lot, but on a microcontroller with 64K every byte counts, and I'm assuming that's why.

Would it not be more appropriate to calculate parity using XORs? This ought to be more efficient CPU and memory-wise. I'd be happy to add that to this PR but I don't want too much scope creep without checking.

Reply:
Honestly I don't know how it came to take such choice, the only thing I'm pretty sure is that we need fast parity check in the proxmark. On the client of course that doesn't matter.
If you're up to make some tests and some profiling to see which approach would be faster on ARM, you're welcome :)